libs/template/config_test.go

```diff
@@ -52,6 +52,15 @@ func TestTemplateConfigAssignValuesFromFileDoesNotOverwriteExistingConfigs(t *te
 	assert.Equal(t, "this-is-not-overwritten", c.values["string_val"])
 }
 
+func TestTemplateConfigAssignValuesFromFileFiltersPropertiesNotInTheSchema(t *testing.T) {
+	c := testConfig(t)
+
+	err := c.assignValuesFromFile("./testdata/config-assign-from-file-unknown-property/config.json")
```

Comment:
can we add some known field to this file to make sure it's set but unknown field is now?